contracts/ownership/Claimable.sol

```diff
@@ -10,30 +10,35 @@ import "./Ownable.sol";
  * This allows the new owner to accept the transfer.
  */
 contract Claimable is Ownable {
-  address public pendingOwner;
+  address private pendingOwner_;
 
   /**
    * @dev Modifier throws if called by any account other than the pendingOwner.
    */
   modifier onlyPendingOwner() {
-    require(msg.sender == pendingOwner);
+    require(msg.sender == pendingOwner_);
     _;
   }
 
+  /**
+   * @return the address of the pending owner.
+   */
+  function pendingOwner() public view returns(address) {
+    return pendingOwner_;
+  }
+
   /**
    * @dev Allows the current owner to set the pendingOwner address.
    * @param newOwner The address to transfer ownership to.
    */
   function transferOwnership(address newOwner) public onlyOwner {
-    pendingOwner = newOwner;
+    pendingOwner_ = newOwner;
   }
 
   /**
    * @dev Allows the pendingOwner address to finalize the transfer.
    */
   function claimOwnership() public onlyPendingOwner {
-    emit OwnershipTransferred(owner, pendingOwner);
-    owner = pendingOwner;
-    pendingOwner = address(0);
```

Comment:
This line should stay because it's not part of `_transferOwnership`. But we've removed `Claimable` in #1274.

Reply:
Ah, #1274 is not merged yet.

Reply:
indeed, and no test failed :-1: